src/dlgTriggerEditor.cpp

```diff
@@ -8426,40 +8426,20 @@ void dlgTriggerEditor::slot_keyGrab()
 // Deactivate instead with optional "false" - to allow these for keybindings
 void dlgTriggerEditor::setShortcuts(const bool active)
 {
-    QList<QAction*> actionList = toolBar->actions();
-    QString actionText;
+    setShortcuts(toolBar->actions(), active);
+    setShortcuts(toolBar2->actions(), active);
+}
+
+void dlgTriggerEditor::setShortcuts(QList<QAction*> actionList, const bool active) {
```

Comment:
Remember we put the opening brace on the next line - even if your IDE doesn't automatically
```suggestion
void dlgTriggerEditor::setShortcuts(QList<QAction*> actionList, const bool active)
{
```